src/screens/NotificationScreen.js

```diff
@@ -1,23 +1,141 @@
+      vaccineName: "COVID-19 Vaccine2",
+      date: "April 15th",
+    },
+    {
+      status: "UPCOMING",
+      name: "Jasper Matthews3",
+      vaccineName: "COVID-19 Vaccine3",
+      date: "April 15th",
+    },
+    {
+      status: "OVERDUE",
+      name: "Jasper Matthews4",
+      vaccineName: "COVID-19 Vaccine4",
+      date: "April 15th",
+    },
+  ];
+
+  const statusUpcoming = () => {
+    return (
+      <View>
+        {notification
+          .filter((item) => item.status === "UPCOMING")
+          .map((item, index) => {
+            return (
+              <ContentItem
+                key={index}
+                name={item.name}
+                vaccineName={item.vaccineName}
+                date={item.date}
+              />
+            );
+          })}
+      </View>
+    );
+  };
+
+  const statusOverdue = () => {
+    return (
+      <View>
+        {notification
+          .filter((item) => item.status === "OVERDUE")
+          .map((item, index) => {
+            return (
+              <ContentItem
+                key={index}
+                name={item.name}
+                vaccineName={item.vaccineName}
+                date={item.date}
+              />
+            );
+          })}
+      </View>
+    );
+  };
+
+  const statusToday = () => {
+    return (
+      <View>
+        {notification
+          .filter((item) => item.status === "TODAY")
+          .map((item, index) => {
+            return (
+              <ContentItem
+                key={index}
+                name={item.name}
+                vaccineName={item.vaccineName}
+                date={item.date}
+              />
+            );
+          })}
+      </View>
+    );
+  };
+
   return (
-    <SafeAreaView style={styles.container}>
-      <Text style={styles.text}>This is notification screen</Text>
+    <SafeAreaView style={screenConfig.container}>
+      <View style={styles.textContainer}>
+        <View style={styles.container}>
+          <Text style={[styles.status, styles.red]}>{"TODAY"}</Text>
+          {statusToday()}
+        </View>
+      </View>
+      <View style={styles.textContainer}>
+        <View style={styles.container}>
+          <Text style={[styles.status, styles.blue]}>{"UPCOMING"}</Text>
+          {statusUpcoming()}
+        </View>
+      </View>
+      <View style={styles.textContainer}>
+        <View style={styles.container}>
+          <Text style={[styles.status, styles.blue]}>{"OVERDUE"}</Text>
+          {statusOverdue()}
+        </View>
+      </View>
     </SafeAreaView>
   );
 };
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: colors.white,
-    flex: 1,
+    marginLeft: scale(15),
+    marginTop: verticalScale(15),
+    marginBottom: verticalScale(15),
+  },
+  textContainer: {
+    backgroundColor: colorConfig.white,
+    width: scale(349),
+    marginTop: verticalScale(10),
+  },
+  status: {
+    fontFamily: "fredokaOne-regular",
+    fontSize: moderateScale(12),
```

Comment:
If you want to make the size scaling by the height size of phone, use verticalScale. We don't know the logic of moderateScale so using moderateScale makes the design slightly (but obviously need to fix) different depend on what device we test or use until we've got the logic of moderateScale. Check ContentItem.js too